packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -91,6 +96,33 @@ export function getBeaconPoolApi({
       await network.gossip.publishVoluntaryExit(voluntaryExit);
     },
 
+    async submitPoolBlsToExecutionChange(blsToExecutionChanges) {
```

Comment:
this api is only available from capella, should throw error if we're not there yet?